src/syrupy/serializers/base.py

```diff
@@ -104,7 +101,7 @@ def write(self, data: "SerializableData", index: int = 0) -> None:
         self.write_snapshot_or_remove_file(snapshot_file, snapshot_name, data)
 
     def post_write(self, data: "SerializableData", index: int = 0) -> None:
-        self._snap_file_hook(index)
+        pass
```

Comment:
Can we think of a use case for the hooks? I.e. an argument for keeping them?

Reply:
We don't use it anymore since each assertion keeps track of it's snapshots, but they could be useful for custom plugins performing actions at different stages of serializing

Reply:
We can leave it for now but I think it's better to add it only when we find a use case. Rather than add functionality that won't be used because we're trying to guess at how the community will use this.